src/main/java/appeng/integration/modules/ic2/IC2PowerSinkAdapter.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of Applied Energistics 2.
+ * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+ *
+ * Applied Energistics 2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Applied Energistics 2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package appeng.integration.modules.ic2;
+
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
+
+import ic2.api.energy.prefab.BasicSink;
+import ic2.api.energy.tile.IEnergyEmitter;
+
+import appeng.api.config.PowerUnits;
+import appeng.tile.powersink.IExternalPowerSink;
+
+
+/**
+ * The real implementation of IC2PowerSink.
+ */
+public class IC2PowerSinkAdapter extends BasicSink implements IC2PowerSink
+{
+
+	private final IExternalPowerSink powerSink;
+
+	private final Set<EnumFacing> validFaces = EnumSet.allOf( EnumFacing.class );
+
+	public IC2PowerSinkAdapter( TileEntity tileEntity, IExternalPowerSink powerSink )
+	{
+		super( tileEntity, 0, Integer.MAX_VALUE );
```

Comment:
Any Voltage that is higher the 13 will be automaticly set into the fix. Which means it gets the Integer.MaxValue as max Energy Input.
So doing Tier 13 would be like Integer.maxValue - 500 or something like that still high enought that it would be never reached. But i would still say: Make it tier 3-4 since packets are again a thing.